cmd/buildah/addcopy.go

```diff
@@ -67,6 +68,7 @@ func applyFlagVars(flags *pflag.FlagSet, opts *addCopyResults) {
 	if err := flags.MarkHidden("cert-dir"); err != nil {
 		panic(fmt.Sprintf("error marking cert-dir as hidden: %v", err))
 	}
+	flags.StringVar(&opts.checksum, "checksum", "", "checksum the source content")
```

Comment:
Maybe ?
```suggestion
	flags.StringVar(&opts.checksum, "checksum", "", "checksum the HTTP source content")
```

Reply:
Good suggestion. It's important to be clear here with the incoming support for git URLs.

Also, there doesn't appear to be an existing mechanism to split the flags for ADD and COPY. --checksum only applies to ADD. The same goes for the manpage -- it has to be in both or a check/lint fails.